### PR TITLE
tbi(ui5-writer): updates for locate reuse lib

### PR DESCRIPTION
**Pull request by cianmSAP**
https://github.com/SAP/open-ux-tools/issues/968

- Moves locate-reuse-lib.js to the webapp\test folder and ensure it is referenced from the flpSandbox.html file.
- Removes locate-reuse-lib.js reference from webapp\index.html file.

### CODE REVIEW

**Review by devinea — CHANGES_REQUESTED** [review]
@cianmSAP 1 change needed. 

Comments
`minor` for the changeset should be ok as there is no breaking change to the API. 
But `resourceroots` would need to be manually added in the `index.html` if the customer needs to load a reuse library later. 
`flpSandbox.html` will still dynamically load it from the frontend server (if needed and available) during local preview. In production the Fiori Launchpad will register the libraries neededs at runtime.

File: packages/ui5-application-writer/templates/core/webapp/index.html [hunk 1]
```diff
@@ -21,11 +21,7 @@
         data-sap-ui-compatVersion="edge"
         data-sap-ui-async="true"
         data-sap-ui-frameOptions="trusted"
```
[the remark this answers is not in the document]
Comment (cianmSAP) [reply packages/ui5-application-writer/templates/core/webapp/index.html:23]: Updated

**Review by devinea — APPROVED** [review]
Thanks @cianmSAP 
Updates to templates and snapshots look good.
I did a quick test with `/packages/fiori-elements-writer/test/test-output/lropTemplates/lropV2` locally and `start` and `start-noflop` work well.

**Review by IainSAP — COMMENTED** [review]
Thx @cianmSAP 

- Paths updated as expected
- Test snapshots updated as expected

**Review by IainSAP — APPROVED** [review]

**Pull request merged**